### PR TITLE
Meta+LibWeb: Update Skia to version 129

### PR DESCRIPTION
**Pull request by kalenikaliaksandr**

### CODE REVIEW

File: Toolchain/BuildVcpkg.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ if [ "$ci" -eq 0 ]; then
 fi
 
 GIT_REPO="https://github.com/microsoft/vcpkg.git"
-GIT_REV="3508985146f1b1d248c67ead13f8f54be5b4f5da" # 2024.08.23
+GIT_REV="a39a74405f277773aba08018bb797cb4a6614d0c" # 2024.09.19
```
Comment (trflynn89) [inline Toolchain/BuildVcpkg.sh:24]: There isn't a September release yet?
<https://github.com/microsoft/vcpkg/releases>

Comment (kalenikaliaksandr) [reply Toolchain/BuildVcpkg.sh:24]: I didn't know that vcpkg has a release cycle and assumed it's fine to use latest commit hash from vcpkg as a baseline 😅

I updated commit message to not mention "release". isn't that ok to use arbitrary commit has from vcpkg as a release? or there are some good reasons to hold to using release versions?

Comment (trflynn89) [reply Toolchain/BuildVcpkg.sh:24]: tbh I don't know! They put out a [whole release page](https://devblogs.microsoft.com/cppblog/whats-new-in-vcpkg-august-2024/) for the official tagged version. But I can't find any info online about whether it's best practice to stick to releases or if it's fine to just pick any hash.

**Review by trflynn89 — APPROVED** [review]
Let's go for it :)

**Pull request merged**